<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1182</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1304</width>
         <height>1048</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="res.qrc">:/imagesPic/pic/ymy_1280x1024.jpg</pixmap>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1182</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <column>
        <property name="text">
         <string>节点</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>节点类型</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>图片文件</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/imagesIcon/icons/Pictures_Picture_Folder_48px_515430_easyicon.net.png</normalon>
         </iconset>
        </property>
        <item>
         <property name="text">
          <string>分组节点</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/imagesIcon/icons/folder_48px_42551_easyicon.net.png</normalon>
          </iconset>
         </property>
         <item>
          <property name="text">
           <string>图片节点</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/imagesIcon/icons/Picture_48px_507712_easyicon.net.png</normalon>
           </iconset>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>分组2</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/imagesIcon/icons/folder_48px_42551_easyicon.net.png</normalon>
          </iconset>
         </property>
         <item>
          <property name="text">
           <string>图片</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/imagesIcon/icons/Picture_48px_507712_easyicon.net.png</normalon>
           </iconset>
          </property>
         </item>
        </item>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actAddFloder">
   <property name="icon">
    <iconset>
     <normaloff>:/images/icons/folder_48px_42551_easyicon.net.png</normaloff>:/images/icons/folder_48px_42551_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>添加目录...</string>
   </property>
   <property name="toolTip">
    <string>添加目录</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actAddFiles">
   <property name="icon">
    <iconset>
     <normaloff>:/images/icons/document_file_48px_563728_easyicon.net.png</normaloff>:/images/icons/document_file_48px_563728_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>添加文件...</string>
   </property>
   <property name="toolTip">
    <string>添加文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actZoomIn">
   <property name="icon">
    <iconset>
     <normaloff>:/images/icons/Magnifier_zoom_in_49.156626506024px_1198331_easyicon.net.png</normaloff>:/images/icons/Magnifier_zoom_in_49.156626506024px_1198331_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="toolTip">
    <string>放大</string>
   </property>
  </action>
  <action name="actZoomOut">
   <property name="icon">
    <iconset>
     <normaloff>:/images/icons/Magnifier_zoom_out_49.156626506024px_1198332_easyicon.net.png</normaloff>:/images/icons/Magnifier_zoom_out_49.156626506024px_1198332_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="toolTip">
    <string>缩小</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actZoomRealSize">
   <property name="icon">
    <iconset>
     <normaloff>:/images/icons/magnify_48.091254752852px_1190644_easyicon.net.png</normaloff>:/images/icons/magnify_48.091254752852px_1190644_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>实际大小</string>
   </property>
   <property name="toolTip">
    <string>实际大小</string>
   </property>
  </action>
  <action name="action">
   <property name="icon">
    <iconset>
     <normaloff>:/images/icons/Width_Window_48px_1179015_easyicon.net.png</normaloff>:/images/icons/Width_Window_48px_1179015_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>适合宽度</string>
   </property>
   <property name="toolTip">
    <string>适合宽度</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actDeleteItem">
   <property name="icon">
    <iconset>
     <normaloff>:/images/icons/delete_exit_remove_48px_3590_easyicon.net.png</normaloff>:/images/icons/delete_exit_remove_48px_3590_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>删除节点</string>
   </property>
   <property name="toolTip">
    <string>删除节点</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset>
     <normaloff>:/images/icons/account_door_exit_logout_39.384615384615px_1211608_easyicon.net.png</normaloff>:/images/icons/account_door_exit_logout_39.384615384615px_1211608_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actZoomFith">
   <property name="icon">
    <iconset>
     <normaloff>:/images/icons/Height_Window_48px_1177939_easyicon.net.png</normaloff>:/images/icons/Height_Window_48px_1177939_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>适合高度</string>
   </property>
   <property name="toolTip">
    <string>适合高度</string>
   </property>
   <property name="shortcut">
    <string>Meta+H</string>
   </property>
  </action>
  <action name="actScanItems">
   <property name="icon">
    <iconset>
     <normaloff>:/images/icons/tool_node_editor_48px_1066386_easyicon.net.png</normaloff>:/images/icons/tool_node_editor_48px_1066386_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>遍历节点</string>
   </property>
   <property name="toolTip">
    <string>遍历节点</string>
   </property>
  </action>
  <action name="actDockVisible">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/images/icons/magnification_48px_1155062_easyicon.net.png</normaloff>:/images/icons/magnification_48px_1155062_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>窗体可见</string>
   </property>
   <property name="toolTip">
    <string>窗体可见</string>
   </property>
  </action>
  <action name="actDockFloat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/images/icons/gps_location_needle_push_pin_thumbtack_48px_166_easyicon.net.png</normaloff>:/images/icons/gps_location_needle_push_pin_thumbtack_48px_166_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>窗体浮动</string>
   </property>
   <property name="toolTip">
    <string>窗体浮动</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
